app/config/config_dev_event_replay.yml

```diff
@@ -3,6 +3,3 @@ imports:
 
 swiftmailer:
     disable_delivery: true
-
-surfnet_stepup_middleware_command_handling:
-    warn_on_missing_email_template: false
```

Comment:
shouldn't this also mean that some configuration should be removed from a bundle?